### PR TITLE
QuantumLeap helm chart to be deployed on OpenShift

### PR DESCRIPTION
**Pull request by ernesgonzalez33**
Quantum leap chart with its documentation to be deployed on OpenShift

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+repos:
+  - repo: https://github.com/norwoodj/helm-docs
```
Comment (jtudelag) [inline .pre-commit-config.yaml:2]: @ernesgonzalez33 @arturohu we need still decide if we want this on the CI or as pre-hook, but nice work!

File: applications/quantumleap/chart/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  - quantumleap
+  - helm
+  - chart
+sources:
+  - https://github.com/FIWARE-Ops/marinera/applications/quantumleap
+dependencies:
+- name: quantumleap
+  version: 0.1.18
+  repository: https://orchestracities.github.io/charts/
+maintainers:
```
Comment (jtudelag) [inline applications/quantumleap/chart/Chart.yaml:17]: Very complete Chart.yaml, my main concern is that we (you Ernesto and Arturo) appear as mantainers...while this is just an umbrella chart consuming the official quantum leap one as a dependency...I mean, not much we are maintaining, we are just consuming. If I were someone new to this, I would think you are the maintainers of the orchestracities chart...

File: applications/quantumleap/chart/values.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
+quantumleap:
+  # Default values for quantumleap.
+  # This is a YAML-formatted file.
+  # Declare variables to be passed into your templates.
+  
+  # -- Number of replicas
+  replicaCount: 2
+  image:
+    # -- Container image to deploy
+    repository: smartsdk/quantumleap
```
Comment (jtudelag) [inline applications/quantumleap/chart/values.yaml:10]: Can set the full image location? sth like docker.io/smartsdk/quantumleap (or wherever this comes from)?

File: applications/quantumleap/chart/Chart.yaml (hunk 2 above)
Comment (ernesgonzalez33) [reply applications/quantumleap/chart/Chart.yaml:17]: We could remove the maintainer part altogether. Makes sense what you say

File: .pre-commit-config.yaml (hunk 1 above)
Comment (ernesgonzalez33) [reply .pre-commit-config.yaml:2]: Should we remove it for the moment?

Comment (jtudelag) [reply .pre-commit-config.yaml:2]: Lest discuss it all together.

File: applications/quantumleap/chart/values.yaml (hunk 3 above)
Comment (ernesgonzalez33) [reply applications/quantumleap/chart/values.yaml]: Done

File: .pre-commit-config.yaml (hunk 1 above)
Comment (wistefan) [reply .pre-commit-config.yaml:2]: I would leave it as a first step and build the ci based on that work in the second step.

**Review by wistefan — APPROVED** [review]
LGTM

**Pull request merged**